infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -419,6 +419,31 @@ func (b *TFBlueprintTest) GetTFSetupStringOutput(key string) string {
 	return terraform.Output(b.t, &terraform.Options{TerraformDir: b.setupDir, Logger: b.logger, NoColor: true}, key)
 }
 
+// GetTFSetupJsonOutput returns TF setup output for a given key as gjson.Result.
+// An empty string for key can be used to return all values.
+// It fails test if given key does not output a primitive, if setupDir is not configured, or invalid JSON.
```

Comment:
I may have misunderstood but couldn't output for a key be any arbitrary json that passes gjson.Parse?

Reply:
Yeah - Bad carry over from `GetTFSetupStringOutput`.